libraries/consul_ui.rb

```diff
@@ -26,7 +26,7 @@ class ConsulUI < Chef::Resource
 
       # @!attribute binary_url
       # @return [String]
-      attribute(:binary_url, kind_of: String, default: 'https://releases.hashicorp.com/consul/%{version}/%{filename}.zip')
+      attribute(:binary_url, kind_of: String, default: 'https://releases.hashicorp.com/consul/%{version}/consul_%{filename}.zip')
```

Comment:
I'm pretty sure merging this PR in combination with #252  will break the cookbook. The default value for :binary_url will return `consul_consul_0.6.0_webui.zip` and that URL doesn't exist. I'd suggest reverting this merge.